src/main/java/yesman/epicfight/api/client/event/types/BuildCameraTransform.java

```diff
@@ -23,9 +23,26 @@ public float getPartialTick() {
 	}
 	
 	public static final class Pre extends BuildCameraTransform implements CancelableEvent {
+		private boolean cancelVanillaCameraSetup = false;
+		
 		public Pre(EpicFightCameraAPI cameraApi, Camera camera, float partialTick) {
 			super(cameraApi, camera, partialTick);
 		}
+		
+		/**
+		 * Set true if you want to disable the vanilla camera setup process
+		 * This means, you have modified the camera transform through this event
+		 * and don't want it to be overwrite by vanilla camera setups.
+		 * 
+		 * @see MixinCamera#epicfight$setup
+		 */
```

Comment:
```suggestion
		/// Set to `true` to disable the vanilla camera setup process.
		/// Indicates that the camera transform has been modified through this event
		/// and should not be overwritten by vanilla camera setups.
		///
		/// @see MixinCamera#epicfight$setup
```

We should keep any new comments in Markdown to ease the migration in the future.

I have also made some other changes to keep the doc comments slightly easier to read and formal, without saying `You need to` or `we have to` or `but this could`.

Reply:
For now, I just kept old comment style in 1.20.1 since I use eclipse and others all use javadoc style comments.

I uses markdown standard only in 1.21.1.

Reply:
> For now, I just kept old comment style in 1.20.1 since I use eclipse and others all use javadoc style comments.

I think that's fine, the doc comment is still readable, and we should not wait for the current JDK to be exactly `23` to make it easier to handle the change.

Does the doc comment not get displayed in Eclipse?